modules/fluid_properties/src/fluidproperties/TabulatedFluidProperties.C

```diff
@@ -609,6 +623,15 @@ TabulatedFluidProperties::h_from_p_T(Real pressure, Real temperature) const
   }
 }
 
+ADReal
+TabulatedFluidProperties::h_from_p_T(const ADReal & pressure, const ADReal & temperature) const
+{
+  if (_fp) // Assuming _fp can handle ADReal types
+    return _fp->h_from_p_T(pressure, temperature);
+  else
+    paramError("fp", "Fluid properties object does not support ADReal computations.");
```

Comment:
cant handle this specific one

maybe create a routine with a comprehensive error message, that says what is missing
TFP::fluidPropertyForwardError(std::string desired_routine, bool is_ad)

Reply:
added in [a6a0e70](https://github.com/idaholab/moose/pull/27506/commits/a6a0e70fd320f10a2453e722ddc0c69452dc09be)